spec/models/list_import/my_anime_list/row_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe ListImport::MyAnimeList::Row do
+  context 'with anime' do
+    let(:anime) { fixture('list_import/my_anime_list/toy-anime.json') }
```

Comment:
This is MAL data tho. Is there a way I can do that without a fixture? I didn't think there was.